src/CppParser/Parser.cpp

```diff
@@ -903,6 +903,14 @@ void Parser::WalkRecord(const clang::RecordDecl* Record, Class* RC)
         ReadClassLayout(RC, Record, CharUnits(), true);
     }
 
+    if (Record->redecls_begin() != Record->redecls_end()) {
```

Comment:
Some formatting here: opening braces must go to their own line, local variables should start with a capital letter (Clang style)